marlowe-spec-test/README.md

```diff
@@ -0,0 +1,205 @@
+# Marlowe Spec Test
+
+This tool is a test suite that checks if a particular Marlowe implementation conforms to the specification.
+
+The tool receives as an argument the path to a process (with no arguments) that we call "client process". The client process listens for semantic requests over `stdin` and replies via `stdout`, any `stderr` message is just printed.
+
+## Usage
+
+**Important Note**
+
+We haven't packaged the tool yet, so in order to run it, you need to be inside the `nix develop` shell and execute it with `cabal run`.
+
+```bash
+# Whenever you see something like
+$ marlowe-spec -c <path/to/cli>
+# For the moment replace it with
+[nix-shell] $ cabal run marlowe-spec -- -c <path/to/cli>
+```
+
+### Get help
+
+To list all available options, you can invoke the tool with `--help`
+
+```bash
+$ marlowe-spec --help
+
+Mmm... tasty test suite
```

Comment:
One of the drawbacks of using Tasty as the runner... I was able to add options to it, but wasn't able to modify this message. At least not easily.